src/components/Categories/LoanSpotlight.vue

```diff
@@ -0,0 +1,255 @@
+<template>
+	<div class="tw-bg-primary md:tw-bg-secondary">
+		<kv-page-container>
+			<kv-grid>
+				<h2 class="md:tw-hidden">
+					Today's loan spotlight
+				</h2>
+				<div class="md:tw-flex md:tw-pt-8 md:tw-pb-8">
+					<div class="md:tw-mr-3 lg:tw-mr-4 md:tw-flex-none">
+						<kv-responsive-image
+							class="spotlight-loan-image"
+							:images="getSpotlightImage"
+							loading="lazy"
+							:alt="altText"
+						/>
+					</div>
+					<div class="md:tw-grow">
+						<h2 class="tw-hidden md:tw-block tw-pt-1">
+							Today's loan spotlight
+						</h2>
+						<h3 class="tw-pt-2 tw-mb-1">
+							{{ getSpotlightLoanLocation }}
+						</h3>
+						<span class="tw-line-clamp-5">
+							<p
+								v-for="(paragraph, index) in getSpotlightText"
+								:key="index"
+								v-html="paragraph"
+							>
+							</p>
+						</span>
+						<div class="tw-mt-2">
+							<kv-button
+								class="tw-w-full md:tw-w-auto"
+								:to="`/lend-beta/${this.spotlightLoan.id}`"
+								variant="primary"
+								state=""
+								v-kv-track-event="['Lending', 'click-loan-spotlight', 'View Loan']"
+							>
+								View loan
+							</kv-button>
+						</div>
+					</div>
+				</div>
+			</kv-grid>
+		</kv-page-container>
+	</div>
+</template>
+
+<script>
+
+import { toParagraphs } from '@/util/loanUtils';
+import gql from 'graphql-tag';
+import logReadQueryError from '@/util/logReadQueryError';
+import KvResponsiveImage from '@/components/Kv/KvResponsiveImage';
+import KvGrid from '~/@kiva/kv-components/vue/KvGrid';
+import KvPageContainer from '~/@kiva/kv-components/vue/KvPageContainer';
+import KvButton from '~/@kiva/kv-components/vue/KvButton';
+
+const routePath = 'recommended-by-lenders';
```

Comment:
Let's make the default category string be a prop  that is passed to this component. This will allow the component to be used to spotlight other categories